pod-compose.sh

```diff
@@ -3,6 +3,7 @@
 set -o allexport
 source .env
 
+mkdir "$HOME/data_server/results" -p
```

Comment:
can we check if it already exists ? and if not then go ahead and mkdir , else will throw an error